src/index.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const state = {
+  tempValue: 80,
+  city: 'Seattle',
+  lat: 0,
+  lon: 0
+};
+
+const temperature = document.getElementById('tempValue');
+
+const increaseTemp = () => {
+  state.tempValue += 1;
+  temperature.textContent = state.tempValue + '°F';
+  updateBackground();
+  updateTempColor();
+};
+
+const decreaseTemp = () => {
+  state.tempValue -= 1;
+  temperature.textContent = state.tempValue + '°F';
+  updateBackground();
+  updateTempColor();
+};
+
+const updateBackground = () => {
+  const background = document.getElementById('bg');
+
+  if (state.tempValue > 100){
+    background.style.backgroundImage = 'url(assets/sun-surface.jpg)';
+  }
+  else if (state.tempValue > 90){
+    background.style.backgroundImage = 'url(assets/desert.jpg)';
+  }
+  else if (state.tempValue > 80){
+    background.style.backgroundImage = 'url(assets/desert1.jpg)';
+  }
+  else if (state.tempValue > 70){
+    background.style.backgroundImage = 'url(assets/summer.jpg)';
+  }
+  else if (state.tempValue > 60){
+    background.style.backgroundImage = 'url(assets/spring.jpg)';
+  }
+  else if (state.tempValue > 50){
+    background.style.backgroundImage = 'url(assets/spring2.webp)';
+  }
+  else if (state.tempValue > 40){
+    background.style.backgroundImage = 'url(assets/autumn.jpg)';
+  }
+  else if (state.tempValue > 30){
+    background.style.backgroundImage = 'url(assets/winter-lanscape.webp)';
+  }
+  else if (state.tempValue > 20){
+    background.style.backgroundImage = 'url(assets/deepfreeze.jpg)';
+  }
+};
+
+const updateTempColor = () => {
+  if (state.tempValue >= 80) {
+    temperature.classList = 'red';
+  } else if (state.tempValue >= 70) {
+    temperature.classList = 'orange';
+  } else if (state.tempValue >= 60) {
+    temperature.classList = 'yellow';
+  } else if (state.tempValue >= 50) {
+    temperature.classList = 'green';
+  } else {
+    temperature.classList = 'teal';
+  }
+};
+
+const updateSky = () => {
+  const sky = document.getElementById('sky');
+  const skySelection = document.getElementById('skySelection').value;
+
+  if (skySelection === 'Sunny'){
+    sky.textContent = '☁️  🕊️☁️        ☀️ ☁️';
+  }
+  else if (skySelection === 'Cloudy'){
+    sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  }
+  else if (skySelection === 'Rainy'){
+    sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧';
+  }
+  else if (skySelection === 'Snowy'){
+    sky.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+};
+
+const getLatLon = () => {
+  return axios
+    .get('http://localhost:5000/location', {
+      params: {
+        q: state.city,
+      },
+    })
+    .then((response) => {
+      state.lat = response.data[0].lat;
+      state.lon = response.data[0].lon;
+    })
+
+    .catch((error) => {
+      console.log('error getting lat and lon');
+    });
+};
+
+const updateCityName = () => {
+  let cityNameInput = document.getElementById('cityNameInput').value;
```

Comment:
both of these variables aren't reassigned, so they could be declared as `const` preferably.